tests/test_discovery.py

```diff
@@ -748,3 +742,106 @@ async def test_discovery_device_repr(discovery_mock, mocker):
         assert "update() needed" not in repr_
     else:
         assert "update() needed" in repr_
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    ("info", "needs_query", "host"),
+    [
+        (
+            {
+                "result": {
+                    "device_type": "IOT.SMARTPLUGSWITCH",
+                    "device_model": "HS100(UK)",
+                    "device_id": "id",
+                    "ip": "127.0.0.1",
+                    "mac": "00-00-00-00-00-00",
+                    "mgt_encrypt_schm": {
```

Comment:
Would it make sense to either move the infos into separate files, or at least to separate variables?